webapp/packages/core-blocks/src/Split/Pane.m.css

```diff
@@ -3,3 +3,8 @@
   overflow: auto;
   z-index: 0;
 }
+
+.hiddenContent {
```

Comment:
This content will be accessible to screen readers

Reply:
moreover elements will be accessible with TAB